pkg/frontend/transport/handler.go

```diff
@@ -204,8 +204,8 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 		"path", r.URL.Path,
 		"response_time", queryResponseTime,
 		"query_wall_time_seconds", wallTime.Seconds(),
-		"query_series", numSeries,
-		"query_bytes", numBytes,
+		"fetched_series_count", numSeries,
+		"fetched_chunk_bytes", numBytes,
```

Comment:
[nit] Should we keep the naming consistent with the metric?
```suggestion
		"fetched_chunks_bytes", numBytes,
```